joomla/joomla-cms
# [#33321] Add labels in some admin pages help us easily choose items.

# Test Instructions

Please go to `option=com_cache` page, you will notice that we have to click the checkbox to check every item, there is no use to click item title.

![p-2014-06-14-4](https://cloud.githubusercontent.com/assets/1639206/3278916/580a11d6-f3dc-11e3-8275-bcc5b67231c1.jpg)

Then we checkout to this PR, now just click item title, the checkbox will be checked.

![p-2014-06-14-5](https://cloud.githubusercontent.com/assets/1639206/3278917/5bd4923c-f3dc-11e3-9161-b8991e330f10.jpg)

This is an UX enhancement update.
# Changed pages:
- Joomla Installer default language page
- Admin Language Manager: Site & Admin list
- Admin Installer: Install language
- Admin Installer: Manage
- Admin Installer: Discover
- Admin Installer: Update

![140221-0002](https://f.cloud.github.com/assets/1639206/2226273/2ec68462-9a9f-11e3-8643-f12e5b4c5720.jpg)
![140221-0003](https://f.cloud.github.com/assets/1639206/2226275/2ec7582e-9a9f-11e3-8c42-a9227b72db8f.jpg)
![140221-0004](https://f.cloud.github.com/assets/1639206/2226274/2ec700d6-9a9f-11e3-8636-5ec4ef33dae5.jpg)
![140221-0005](https://f.cloud.github.com/assets/1639206/2226276/2ec90a66-9a9f-11e3-8ba1-e3b03fa98b42.jpg)
![140221-0006](https://f.cloud.github.com/assets/1639206/2226277/2ecae070-9a9f-11e3-9599-c3a953fccd4d.jpg)
![140221-0007](https://f.cloud.github.com/assets/1639206/2226279/3fc73d24-9a9f-11e3-987a-2321a008c7f2.jpg)

Tracker: http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_item_id=33321&start=0
